README.md

```diff
@@ -13,6 +13,16 @@ See the [documentation](https://appsody.dev/docs) for more details:
 * [Installation](https://appsody.dev/docs/getting-started/installation)
 * [CLI Reference](https://appsody.dev/docs/using-appsody/cli-commands)
 
+For information on building the Appsody CLI from source, see [Building from Source](build.md)
 
+## Contributing
 
-For information on building the Appsody CLI from source, see [Building from Source](build.md)
\ No newline at end of file
+We welcome all contributions to the Appsody project. Please see our [Contributing guidelines](https://github.com/appsody/docs/blob/master/CONTRIBUTING.md)
+
+## Community
```

Comment:
The stacks repo calls this "Need Help?" with a bit different text. Lets pick one or the other and be consistent.

Reply:
yep, I'll change this to `Community` across all repos